reference/deprecation-notices.md

```diff
@@ -3,6 +3,18 @@
 
 This document contains a list of deprecation notices for Anbox Cloud and its components.
 
+## Kernel 6.8 and earlier
+*Deprecated in 1.25.0* ; *Removal in 1.27.0*
+
+The support for kernel 6.8 and earlier is deprecated as of 1.25.0. Starting from 1.27.0, Anbox Cloud will run only on kernels with version later than 6.8.
+
+## LXD 5.0
+*Deprecated in 1.25.0* ; *Removal in 1.27.0*
```

Comment:
AFAICT from the [spec](https://docs.google.com/document/d/1HXJr020rhwjj0AM18SYZmIBX4kTp-HehWZNfgbqcJq0/edit?tab=t.0), this should be 1.28.0:
```suggestion
*Deprecated in 1.25.0* ; *Removal in 1.28.0*
```